### PR TITLE
HelloMicroverse/Linters-Setup-Task

### PR DESCRIPTION
**Pull request by RichardChileya**
In this section we: 
Created a new branch according to GitHub flow rules
Added a .gitignore file
setup a linters for html and css and made sure that files not relevant are ignored by git.
Added a descriptive README file 
Add 3 files (each file in a separate commit)
Added an HTML file which includes a header with the text " Hello Microverse"
Added a CSS file with a design to any element
Added a test.md file which will be ignored by git

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,32 @@
...
-Hello Microverse Task
+
+# Project Name: Hello Microverse
+
+> Hello Microverse Task 
+
+## Authors
+
+👤 **Richard W.K Chileya**
+
```
Comment (koachcode) [inline README.md]: - Good work so far, but there is an issue with your readme, Please make it more professional, presently it is missing things like the languages used, please make use of this [template](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/documentation/hello_microverse_project.md) and add what is needed for this project.

File: README.md [hunk 2]
```diff
@@ -1,2 +1,32 @@
-# HelloMicroverse1
-Hello Microverse Task
+
+# Project Name: Hello Microverse
+
+> Hello Microverse Task 
+
```
Comment (koachcode) [inline README.md]: - Please make your PR summary more descriptive, as it currently only contains `Add linters in github/workflows` 

**Hint**

- Create a new branch according to the GitHub flow rules.
- Added a .gitignore file.
- Set up a linter for HTML and CSS and made sure that any files that are not relevant are ignored by git.
- Added a descriptive README file to your project .
- Add 3 files (each one in a separate commit):
- Added a.html file that includes a header with the text "Hello Microverse!". Use h1 tag for that.
- Added a .css file that includes one class with styles for your header (make it in your favorite color).
- A test.md file with the text "This file should be ignored by git".

**Review by koachcode — CHANGES_REQUESTED** [review]
Hi @RichardChileya , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
you-are-near

## Highlights
- NO linter errors :+1: 
- Good commit messages :+1: 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md (hunk 2 above)
Comment (RichardChileya) [reply README.md]: Thank you so much... would you suggest that i add the content you've sent in your review comment ?

**Review by sumairq — APPROVED** [review]
### Hi @RichardChileya  , 



Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
✔️ Good commit history
✔️Linters are passing
✔️Very professional README
✔️Correct Github flow
✔️Descriptive PR
## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**